package.json

```diff
@@ -36,11 +36,8 @@
     ],
     "nohoist": [
       "@celo/verifier/react-native",
```

Comment:
I think we remove this too now that we are always keeping the apps on the same RN version

Reply:
I can try, but @cmcewen told me `react-native` had to be in the package's `node_modules` for metro bundler to work correctly.

Reply:
Ok nvm